scss/templates/_connection-landing.scss

```diff
@@ -11,22 +11,6 @@ $j-gutter: em(20px);
 $landing-bdr: #D9D9D9;
```

Comment:
With the new connection landing page, I'm sure we'll be able to remove most, if not all, "old" connection scss (this entire file)